docker/Dockerfile

```diff
@@ -1,38 +1,31 @@
-FROM ubuntu:xenial
+ARG PHP_VERSION=7.0-apache
+FROM php:${PHP_VERSION}
```

Comment:
What do you think if we do this here instead:
```
ARG PHP_VERSION=7.0
FROM php:${PHP_VERSION}-apache
```

...because we're not really supporting any other types of images but `-apache` so manually typing it in everywhere doesn't make sense?

If this would just be a version number, it would be compatible with existing configs and commands (i.e. `DOCKER_PHP_VERSION=5 yarn docker:up`) if we ever decide to change the way PHP gets installed.